multiqc/modules/preseq/preseq.py

```diff
@@ -70,23 +74,114 @@ def parse_preseq_logs(self, f):
             if float(s[1]) == 0 and float(s[0]) > 0:
                 continue
             data[float(s[0])] = float(s[1])
-            self.total_max = max(float(s[1]), self.total_max)
+
+        # Convert counts to coverage
+        genome_size = getattr(config, 'preseq', {}).get('genome_size')
+        if genome_size is not None:
+            try:
+                genome_size = float(genome_size)
+            except ValueError:
+                genome_sizes = {'hg19': 2897310462,
+                                'hg38': 3049315783,
+                                'mm10': 2652783500}
+                if genome_size in genome_size:
```

Comment:
Yep, typo, thanks! I was always thinking it might be a good idea to avoid variable names that are different only in plural "-s" - this causes typos too often...